commands/push.js

```diff
@@ -69,8 +69,8 @@ let push = async function (context, heroku) {
     process.exit(1)
   }
 
-  let flagsArg = context.flags.arg;
-  let buildArg = (flagsArg !== undefined) ? flagsArg.split(',') : []
+  let flagsBuildArg = context.flags['build-arg'];
+  let buildArg = (flagsBuildArg !== undefined) ? flagsBuildArg.split(',') : []
```

Comment:
Does Docker also support this splitting?

Reply:
No. It requires passing the argument multiple times. I'm not sure how that'd be doable in node though.

Reply:
Looks like the CLI [doesn't handle it at all](https://github.com/heroku/cli-engine-command/blob/master/src/parser.js#L57).

Reply:
@heroku/cli, would this be difficult to support?

Reply:
yes but I actually have a branch that does this, not sure if or when it will land though

Reply:
@jdxcode, it would be great if it could. It seems preferable to using commas as separators because that prevents using commas in values.

Reply:
I'll see if we can track this for next quarter. I abstracted our flag parsing into [this utility](https://github.com/heroku/cli-flags) which supports a multiple: true flag. So we just need to integrate it (which we want to do anyways)